lte/gateway/fabfile.py

```diff
@@ -12,20 +12,28 @@
 """
 
 import sys
-from datetime import datetime
 from time import sleep
 
-from fabric.api import cd, env, execute, lcd, local, run, settings, sudo
-from fabric.contrib.files import exists
-from fabric.operations import get
-from fabric.utils import error, fastprint, puts
+# import fab tasks from dev_tools, so they can be called via fab in the command line
+from dev_tools import (  # noqa: F401
+    check_agw_cloud_connectivity,
+    check_agw_feg_connectivity,
+    deregister_agw,
+    deregister_federated_agw,
+    deregister_feg_gw,
+    register_federated_vm,
+    register_feg_gw,
+    register_vm,
+    register_vm_remote,
+)
+from fabric import Connection, task
 
 sys.path.append('../../orc8r')
 import tools.fab.pkg as pkg
+# import fab tasks from dev_tools, so they can be called via fab in the command line
+from dev_tools import *
```

Comment:
Why is this needed if we already have a detailed import on line 18?

Reply:
That was a leftover/bug from resolving a merge conflict. Removed, since the detailed import of the individual functions suffice.